include/extractor/guidance/validation_handler.hpp

```diff
@@ -46,7 +46,8 @@ class ValidationHandler final : public IntersectionHandler
                             const EdgeID via_eid,
                             Intersection intersection) const override final
     {
-        checkForSharpTurnsOntoRamps(nid, via_eid, intersection);
+        // checkForSharpTurnsOntoRamps(nid, via_eid, intersection);
```

Comment:
this should probably be left active, now that we know our check works. @daniel-j-h any thoughts on applying categories here to distinguish between the handlers results? Or simply keep for now, as both consider ramps.